build/relsignDSCModules.sh

```diff
@@ -155,10 +155,10 @@ if ! unzip -Z 1>/dev/null; then
 fi
 
 # Working directory used for merging both x64 and x86 versions of a DSC module.
-DscModuleIntermediateDir="/tmp/merging_dsc_modules"
+DscModuleIntermediateDir="tmp/merging_dsc_modules"
```

Comment:
Intent was to implement the script in such a way it can be run from any location. The output is created under tmp in script's current location. Is there any extra cleanup step I need to have here for nuke/clean or it will be picked up automatically?